README.md

```diff
@@ -555,15 +555,14 @@ app.launch(function(request,response) {
   response.say("Hello World");
 });
 
-// this call defines a post() and optionally a get() handler in express, mapped to the alexa-app
-// express_app: the express app instance to map to
-//        path: the path prefix to map to
-// enableDebug: when false, don't map a GET handler, default is true
-//              debugging GET requests call express' render() method using 'test'
-app.express(express_app, "/echo/", false);
-
-// now POST calls to /echo/sample in express will be handled by the app.request() function
+// ALWAYS setup the alexa app and attach it to express before anything else.
+app.express({ expressApp: express_app, router: express.Router() });
```

Comment:
because app is too vague as it is :(  This looks less clear what is happening:

```javascript
app.express({ app: express_app, router: express.Router() });
```

`router` is short and it's distinct (express is the only entity that has a router construct in this context.)